samples/ingestion/ingestion-client/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -429,7 +439,13 @@ private async Task ProcessSucceededTranscriptionAsync(string transcriptionLocati
                 var jsonFileName = $"{fileName}.json";
                 var archiveFileLocation = System.IO.Path.GetFileNameWithoutExtension(fileName);
 
-                await StorageConnectorInstance.WriteTextFileToBlobAsync(editedTranscriptionResultJson, FetchTranscriptionEnvironmentVariables.JsonResultOutputContainer, jsonFileName, log).ConfigureAwait(false);
+                var jsonFileUrl = await StorageConnectorInstance.WriteTextFileToBlobAsync(editedTranscriptionResultJson, FetchTranscriptionEnvironmentVariables.JsonResultOutputContainer, jsonFileName, log).ConfigureAwait(false);
+
+                if (!string.IsNullOrEmpty(FetchTranscriptionEnvironmentVariables.CompletedServiceBusConnectionString))
+                {
+                    var completedMessage = new CompletedMessage(audioFileInfo.FileUrl, jsonFileUrl);
+                    await ServiceBusUtilities.SendServiceBusMessageAsync(this.completedServiceBusSender, completedMessage.CreateMessageString(), log, GetMessageDelayTime(serviceBusMessage.PollingCounter)).ConfigureAwait(false);
+                }
```

Comment:
One important thing to note here is that the notification is sent for each audio file that is transcribed, this is not per-batch. I'd love to hear your thoughts/concerns on this - would you rather send this notification per batch, and why?

Reply:
@HenryvanderVegte and I discussed offline that per-batch notification is better since it will result in generating fewer messages on the queue, and it pushes the choice onto the consumer of the message whether to process per-batch or per-file.